app/src/main/java/co/techmagic/hr/presentation/ui/manager/quotes/QuotesManagerImpl.kt

```diff
@@ -0,0 +1,42 @@
+package co.techmagic.hr.presentation.ui.manager.quotes
+
+import android.content.Context
+import co.techmagic.hr.R
+import java.util.*
+
+
+class QuotesManagerImpl(val context: Context) : QuotesManager {
+
+    companion object {
+        private const val QUOTE_TEXT_POSITION = 0
+        private const val QUOTE_AUTHOR_POSITION = 1
+    }
+
+    private val quotes: List<Pair<String, String>>// first - quot, second - author
```

Comment:
quot -> quote
Maybe it worth to create a model or typealias for this to avoid comment

Reply:
Done